src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -158,32 +193,107 @@ export const AdministrationPage = () => {
                             labelOff="Maintenance Mode Off"
                             isChecked={isMaintenanceModeOn}
                             onChange={() => {
+                              setAnnouncementTouched(false);
+                              isMaintenanceModeOn && setEtaTime(undefined);
                               setIsMaintenanceModeOn(!isMaintenanceModeOn);
                             }}
                           />
                         </div>
                       </FormGroup>
                     </GridItem>
                     <GridItem span={12}>
-                      <FormGroup label="Announcement" fieldId="form-announcement">
+                      <FormGroup
+                        label={(isMaintenanceModeOn ? '* ' : '') + 'Announcement'}
+                        fieldId="form-announcement"
+                        helperTextInvalid={invalidAnnouncement() ? 'Required field.' : null}
+                        helperTextInvalidIcon={<ExclamationCircleIcon />}
+                        validated={'error'}
+                      >
                         <TextArea
                           name="form-announcement"
                           id="form-announcement"
                           value={announcementMessage}
                           onChange={(value: string) => {
                             setAnnouncementMessage(value);
                           }}
+                          onBlur={() => {
+                            setAnnouncementTouched(true);
+                          }}
                         />
                       </FormGroup>
                     </GridItem>
+                    {isMaintenanceModeOn && (
+                      <GridItem span={12}>
+                        <FormGroup
+                          label="* ETA Time"
+                          fieldId="eta-time"
+                          helperTextInvalid={invalidEtaTime() ? 'Required field.' : null}
+                          helperTextInvalidIcon={<ExclamationCircleIcon />}
+                          validated={'error'}
+                        >
+                          <DatePicker
+                            required
+                            isDisabled={!isMaintenanceModeOn || isEtaNa}
+                            name="form-etaTime"
+                            id="eta-time"
+                            placeholder="yyyy-MM-dd hh:mm (UTC)"
+                            dateFormat={transDateFormat}
```

Comment:
I haven't found a better way to implement this since no format would look like YYYY-MM-DD HH:MM:SS

Reply:
YYYY-MM-DD HH:MM:SS might be the easiest way for the user to input, here the date format will cause when the user chooses any specific date with the date picker, it will automatically display the date as format defined here. If we instead ask the user to input a date with a format like `Dec 1, 2017, 2:17 PM`, there will be more room for mistakes by missing PM/AM or using blank space instead of `,`. I would suggest we actually use a format that is easiest to input and widely adopted by most of our users.

Reply:
yes, I agree, thanks for this point, let's keep it as it is

Reply:
fixed accordingly